curp/src/members.rs

```diff
@@ -227,6 +258,23 @@ impl ClusterInfo {
         self.cluster_id
     }
 
+    /// Get cluster version
+    #[must_use]
+    #[inline]
+    pub fn cluster_version(&self) -> u64 {
+        self.cluster_version.load(Ordering::Relaxed)
+    }
+
+    /// cluster version increase
+    pub(crate) fn cluster_version_inc(&self) -> u64 {
+        self.cluster_version.fetch_add(1, Ordering::Relaxed)
+    }
+
+    /// cluster version decrease
+    pub(crate) fn cluster_version_dec(&self) -> u64 {
+        self.cluster_version.fetch_sub(1, Ordering::Relaxed)
+    }
```

Comment:
Will these be used across multiple threads? Maybe these also need stricter ordering.

Reply:
Fixed

Reply:
I don't think so. While atomic operations using relaxed memory ordering do not provide any happens-before relationship, they do guarantee a total modification order of each individual atomic variable. This means that all modifications of the same atomic variable happen in an order that is the same from the perspective of every single thread. Do we really need the guarantee of the happens-before relationship for this cluster_version?